### DISCUSSION TITLE
self compile issues

### DISCUSSION THREAD

**Discussion by Jyhade101** [Jyers/Marlin#363] [Q&A] [answered]
hi guys, I'm new here and new to all of the self compiling but have been working on this for over a week, have checked all the wikis and YT vids that even seemed remotely accurate but every time I compile I get errors when compiling

trying to compile the config.h for no probe no runout sensor and i get 
static assertion failed: ADVANCED_PAUSE_FEATURE is required to use M600 with filament runout sensor
 
I'm likely missing something but am at my wits end, please help?

I love the firmware but I'm trying to modify it so that i can set the y axis offset for my hero me gen 5 setup on my e3v2.

**Comment by Jyers** [comment, marked as answer]
First off, welcome to the community! Glad to have you :)
As far as the issue, you need to replace the default configuration files in the marlin folder with one of the sets from `Configuration Files\E3V2 Templates`  Default-NoProbe should be fine for you, but if you want manual mesh leveling, there are also the ManualMesh and UBL-NoProbe configurations which you may find interesting.

**Reply by Jyhade101** [reply]
I tried this and still get errors

**Reply by Jyers** [reply]
The same errors or something new? Make sure you are copying both configuration.h and configuration_adv.h to replace the ones in the marlin folder

**Reply by Cal-LF** [reply]
I have gotten that error also. The filament sensor is enabled default. I have found the Advanced Pause in Configuration_adv.h is not selected default. If I deselect filament sensor it throws the error. Works ok if Advance Pause is selected and leave filament sensor selected also.  Select FILAMENT_LOAD_UNLOAD_GCODES also at the bottom of Advance Paused section.

**Reply by Jyhade101** [reply]
(facepalm) now I feel epicly brilliant. I was placing it in the wrong marlin folder the whole time. thank you for helping me. successfully compiled

**Reply by Cal-LF** [reply]
Jyhade101 - I found it works better for me if I make one change and compile. If it throws errors, I change that selection back. Keeps me from getting lost trying to remember what changes are working or not. ( I am old ).

**Reply by ottobjorkland** [reply]
> First off, welcome to the community! Glad to have you :)
> As far as the issue, you need to replace the default configuration files in the marlin folder with one of the sets from `Configuration Files\E3V2 Templates` Default-NoProbe should be fine for you, but if you want manual mesh leveling, there are also the ManualMesh and UBL-NoProbe configurations which you may find interesting.

Thank you so much! This fixed my newbie problems!
